source/MaterialXCore/Node.cpp

```diff
@@ -20,6 +20,27 @@ const string Backdrop::HEIGHT_ATTRIBUTE = "height";
 //
 // Node methods
 //
+NodePtr GraphElement::addNode(const string& category,
+    const string& name ,
+    const string& type )
+{
+    if (name == "1")
```

Comment:
Why are you checking if the name is "1" and why does the error message for the exception look like the category exception?